components/molecule/photoUploader/src/index.js

```diff
@@ -116,7 +117,7 @@ const MoleculePhotoUploader = forwardRef(
       }
     })
 
-    const _callbackPhotosUploaded = list => {
+    const _callbackPhotosUploaded = (list, data) => {
```

Comment:
Not sure about the `data` naming, it was harder for me to find the proper name for it, `data` is an object with info why occurred the callback _(maybe fallback (?) but it's a contradiction)_, if you have any suggestions, tell me! Thank you!

Reply:
Since it is generic depending the action, `data` looks good to me.